package.json

```diff
@@ -11,12 +13,16 @@
     "nyc": "^11.4.1"
   },
   "scripts": {
-    "test": "etc/prepare-tests.sh && nyc mocha **/*.spec.js",
-    "test-report": "etc/prepare-tests.sh && nyc mocha -R xunit **/*.spec.js > build/test-reports/synctos.xml"
+    "test": "etc/prepare-tests.sh && nyc mocha \"**/*.spec.js\"",
+    "test-report": "etc/prepare-tests.sh && nyc mocha -R xunit \"**/*.spec.js\" > build/test-reports/synctos.xml"
```

Comment:
This change allows Mocha to recurse into subdirectories (e.g. the new `src/validation` directory).